src/annotations/core/AnnotationArea.cpp

```diff
@@ -306,8 +312,6 @@ void AnnotationArea::mouseDoubleClickEvent(QGraphicsSceneMouseEvent *event)
 
 void AnnotationArea::contextMenuEvent(QGraphicsSceneContextMenuEvent *event)
 {
-	finishDrawingItem(event->scenePos());
```

Comment:
Any specific reason why this was removed? This was a workaround for the case when you click right mouse while drawing an item, that caused strange behavior.

Reply:
Oh, sorry, this was a merge error or something; will add it back.